qemu/tests/transfer_file_over_ipv6.py

```diff
@@ -134,14 +134,20 @@ def get_file_md5sum(file_name, session, timeout):
                         error_context.context("Transferring data from %s to %s" %
                                               (vm_src.name, vm_dst.name),
                                               test.log.info)
+                        if params.get("using_guest_interface") == "yes":
+                            src_zone_id = host_ifname
+                            dst_zone_id = inet_name[vm_src]
+                        else:
+                            src_zone_id = host_ifname
```

Comment:
`src_zone_id` is always equal to `host_ifname`, not need to set here.

Reply:
Updated